public/app/plugins/datasource/influxdb/components/ConfigEditor.tsx

```diff
@@ -95,25 +86,14 @@ export class ConfigEditor extends PureComponent<Props> {
         </div>
         <br />
 
-        <h3 className="page-heading">Connection</h3>
-        <div className="gf-form-inline">
-          <div className="gf-form">
-            <InlineFormLabel
-              className="width-10"
-              tooltip="This URL needs to be accessible from the grafana backend/server."
-            >
-              URL
-            </InlineFormLabel>
-            <div className="width-20">
-              <Input
-                className="width-20"
-                value={options.url || ''}
-                placeholder="http://localhost:9999"
-                onChange={this.onUpdateInflux2xURL}
-              />
-            </div>
-          </div>
-        </div>
+        <DataSourceHttpSettings
```

Comment:
How is this actually used?  I expect it would need to be passed to https://github.com/influxdata/influxdb-client-go

Reply:
We create the http client the same way for flux and then pass it to the client lib here https://github.com/grafana/grafana/blob/a2a6b9448ff0516a50738ac53fea28a3365d016f/pkg/tsdb/influxdb/flux/flux.go#L84. So I assume all the standard HTTP settings for the transport are respected.